backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/Migrations.java

```diff
@@ -0,0 +1,38 @@
+package org.molgenis.emx2.sql;
+
+import java.io.IOException;
+import org.molgenis.emx2.MolgenisException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Migrations {
+  private static Logger logger = LoggerFactory.getLogger(Migrations.class);
+
+  public static void initOrMigrate(SqlDatabase db) {
+    int version = MetadataUtils.getVersion(db.getJooq());
+
+    // migration steps
+    if (version < 0) MetadataUtils.init(db.getJooq());
+    if (version < 1)
+      executeMigrationFile(db, "migration1.sql", "upgraded MOLGENIS.version_metadata");
```

Comment:
should probably log the result of a succesfull migration to some kind to update only log table in the db , that way record is kept of the migrations that have been run ( and the order )

Reply:
See previous comment: The way I meant it to work, migrations will always run and always in same order.